### PR TITLE
Add command, command options and test bot data

### PR DESCRIPTION
**Pull request by MikeMcQuaid**
These new categories are available from https://github.com/Homebrew/brew/pull/16847

There's a few TODO here that I'd love help with but don't need to block getting this data flowing.

### CODE REVIEW

File: cmd/formula-analytics.rb [hunk 1]
```diff
@@ -189,6 +219,24 @@ def influx_analytics(args)
...
+          command_and_package, options = tags["command"].split.partition { |arg| !arg.start_with?("-") }
+
+          # Cleanup bad data before https://github.com/Homebrew/homebrew-test-bot/pull/1043
+          # TODO: actually delete this from InfluxDB.
+          # Can delete this code after 27th April 2025.
+          next if %w[audit install linkage style test].exclude?(command_and_package.first)
+          next if command_and_package.last.include?("/")
+          next if options.include?("--tap=")
+          next if options.include?("--only-dependencies")
+          next if options.include?("--cached")
```
Comment (MikeMcQuaid) [inline cmd/formula-analytics.rb:234]: @Bo98 @SMillerDev could do with some help figuring out if we can just delete the data here. I tried and failed:

```console
$ influx delete --bucket analytics --predicate '_measurement="test_bot_test"' --start='2024-04-01T00:00:00Z' --stop='2024-04-26T07:07:19.000Z' --org-id *** --host '***' --token="***"
```

This ran successfully but didn't seem to delete anything 🤷🏻. Help?

File: cmd/formula-analytics.rb [hunk 2]
```diff
@@ -229,7 +276,15 @@ def influx_analytics(args)
...
             nil
           end
+
+          next if count <= 0
+
+          count
         end
+
+        # TODO: we don't seem to get a valid count for these categories, unclear why.
+        count ||= 1 if count_being_weird_categories.include?(category)
```
Comment (MikeMcQuaid) [inline cmd/formula-analytics.rb:286]: @SMillerDev @Bo98 (and any other @homebrew/maintainers who understand InfluxDB): similarly: it'd be good to figure this out. I'm actually not convinced the counts are correct in this case (so am not going to have the data displayed publicly anywhere yet) but hopefully this being merged gives you enough to reproduce the weirdness with some `puts` or `pp`.

Comment (MikeMcQuaid) [reply cmd/formula-analytics.rb:286]: @Bo98 Ah, makes sense now, thanks. Yeh, I think moving this to SQL would be desirable on multiple counts.

File: cmd/formula-analytics.rb (hunk 1 above)
Comment (Bo98) [reply cmd/formula-analytics.rb:234]: Screenshot 2024-05-02 at 19 54 08

Comment (Bo98) [reply cmd/formula-analytics.rb:234]: The CLI as a whole doesn't really work with InfluxDB v3.

**Pull request merged**